manifests/script-lab-react-alpha.outlook.xml

```diff
@@ -184,129 +183,779 @@
         </Resources>
         <!-- Version Overrides 1.1-->
         <VersionOverrides xmlns="http://schemas.microsoft.com/office/mailappversionoverrides/1.1" xsi:type="VersionOverridesV1_1">
-            <Requirements>
-                <bt:Sets DefaultMinVersion="1.3">
-                    <bt:Set Name="Mailbox"/>
-                </bt:Sets>
-            </Requirements>
-            <Hosts>
-                <Host xsi:type="MailHost">
-                    <DesktopFormFactor>
-                        <SupportsSharedFolders>true</SupportsSharedFolders>
-                        <ExtensionPoint xsi:type="AppointmentAttendeeCommandSurface">
```

Comment:
It seems like you removed support for appointment extension points (AppointmentAttendeeCommandSurface, AppointmentOrganizerCommandSurface)...?

Reply:
@ElizabethSamuel-MSFT that's quite possible. Let me check. These manifests get pretty huge and mistakes can happen

Reply:
@ElizabethSamuel-MSFT I checked and AppointmentAttendeeCommandSurface, AppointmentOrganizerCommandSurface are indeed still in the manifests.  Are you saying they don't work as expected?

Reply:
The diff made it seem like those extension points had been removed. I opened "View file" and saw they had not been deleted. Thanks.

Reply:
@ElizabethSamuel-MSFT Yeah this is weird because the diff does indeed indicate that but my local changes do not.  Need to see if something failed to commit properly

Reply:
@ElizabethSamuel-MSFT OK I figured out the problem.  Will ensure everything works locally and push a new commit